Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl

```diff
@@ -106,7 +106,7 @@ Scope(\_SB.PC00) {
         PMEC, 8 //Note: SerialIO Does Not Support PME
       }
       Store(0x3, PMEC) // Set BIT[1:0] = 11b - Power State D3
-      Store(PMEC,Local3) // perform a read to avoid ordering and noncoherency problems
+      //Store(PMEC,Local3) // perform a read to avoid ordering and noncoherency problems
```

Comment:
the comment says it is required to read back.  Since it is hardware requirement, we should follow the requirement.

Reply:
Submitted patch to https://github.com/slimbootloader/slimbootloader/pull/1432